app/decode/ternary.js

```diff
@@ -5,13 +5,32 @@ export const TernaryChars = {
   zero: '0',
   one: '1',
   two: '2',
+  unknown: '?',
 }
 
+export const alphabetVariants = [
+  {
+    label: 'A=0, bez Ch',
+    alphabet: ('abcdefghijklmnopqrstuvwxyz' + TernaryChars.unknown).split(''),
```

Comment:
Je ten 27. znak je spíš neznámý, a nebo spíš mezera/oddělovač? Já vlastně nevím, asi jak kdy. Minimálně v kontextu poláka je to často mezera. Aktuálně řadíme výsledky dle počtu "neznámých" znaků, což by mohlo některá správná řešení upozaďovat.

Reply:
To mi přijde podobné jako Ch v morseovce. Mnedle to je ve většině případů chyba a je dobře, že tyto varianty dáváme dolu. V pár případech to je použito jako mezera, tečka nebo něco speciálního a pak to penalizuje validní odpovědi. Můžeme udělat něco mezi znakem a jasnou chybou a penalizovat to méně.

Reply:
Další varianta je tam dát mezeru a pak mezeru, Q a W penalizovat (méně než nevalidní znak).